### DISCUSSION TITLE
Option to freeze clown crates in place in submarine editor

### DISCUSSION THREAD

**Discussion by TwingoI** [FakeFishGames/Barotrauma#11928] [Ideas and suggestions]
As the title says. This could be used to make hiding spots on submarines which would be perfect for traitor and role-play games.
Also it doesn't have to be just clown crates, it could be any of the crate types and even normal items. This would greatly expand the ability to decorate submarines.

**Comment by abaabaabaa707** [comment]
Would be incredibly useful, greatly increases the use case for the crate and allows for hide and seek outposts.
